src/components/Select/Select.tsx

```
@@ -356,19 +360,29 @@ const SelectTrigger = forwardRef<HTMLButtonElement, SelectTriggerProps>(
-            ? 'cursor-not-allowed text-text-400 pointer-events-none opacity-50'
-            : 'cursor-pointer hover:bg-background-50 focus:bg-accent focus:text-accent-foreground hover:bg-accent hover:text-accent-foreground'
-        }
-        ${!invalid && !disabled ? 'text-text-700' : ''}
-        ${variantClasses}
-        ${className}
-      `}
+        className={cn(
+          'flex w-full items-center justify-between border-border-300',
+          heightClasses,
+          paddingClasses,
+          invalid && `${variant == 'underlined' ? 'border-b-2' : 'border-2'} border-indicator-error text-text-600`,
```

ESLint

[error] Insert `⏎···········`

(prettier/prettier)